src/pyJiraCli/cmd_get_sprints.py

```diff
@@ -74,35 +76,47 @@ def register(subparser) -> argparse.ArgumentParser:
                                         help="The board for which the sprints shall be stored.")
 
     sub_parser_get_sprints.add_argument('--file',
-                                   type=str,
-                                   metavar='<path to file>',
-                                   help="Absolute file path or filepath relativ " + \
-                                        "to the current working directory. " + \
+                                        type=str,
+                                        metavar='<path to file>',
+                                        help="Absolute file path or filepath relativ " +
+                                        "to the current working directory. " +
                                         "The file format must be JSON. ")
 
     return sub_parser_get_sprints
 
-def execute(args) -> Ret.CODE:
+
+def execute(args, server: Server) -> Ret.CODE:
     """ This function servers as entry point for the command 'print'.
         It will be stored as callback for this moduls subparser command.
-    
+
     Args: 
         args (obj): The command line arguments.
-        
+        server (Server): The server object to interact with the Jira server.
+
     Returns:
         Ret:   Returns Ret.CODE.RET_OK if successful or else the corresponding error code.
     """
-    return _cmd_get_sprints(args.board, args.profile, args.file)
+    ret_status = Ret.CODE.RET_ERROR
+
+    # pylint: disable=R0801
+    if server is None:
+        LOG.print_error(
+            "Connection to server is not established. Please login first.")
```

Comment:
print_error needs a ErrorType and Ret.CODE, print_info takes strings as argument